be/src/olap/olap_server.cpp

```diff
@@ -118,6 +118,13 @@ Status StorageEngine::start_bg_threads() {
         LOG(INFO) << "path scan/gc threads started. number:" << get_stores().size();
     }
 
+    // stream load record clean thread
+    RETURN_IF_ERROR(Thread::create(
```

Comment:
No need to create a thread to gc expired data, you can create a Compaction-Filter on this rocksdb instance.
https://github.com/facebook/rocksdb/wiki/Compaction-Filter